packages/DatePicker/src/DatePicker.js

```diff
@@ -37,8 +40,9 @@ const propTypes = {
 
 const defaultProps = {
   children: null,
-  date: null,
   dataFormat: "MM/DD/YYYY",
+  date: null,
+  humanFormat: undefined,
```

Comment:
Can this be `null` instead?

Reply:
Or better, yet, `"MMMM D, YYYY"`, so that you don't have issues with line 69 below...?

Reply:
The actual default value is `"MMMM D, YYYY".  However, it depends on translation because this format still needs to be localized. My thought is if the consumer didn't pass in, I will use 
`I18n.t("datePicker.confirmation_format")`, I did it here:
https://github.com/acl-services/paprika/pull/94/files/e1fc2399b1b667f06cadc0a38e6b0aa2c8e9f533#diff-29403a20e6a7b79ffb881da8212b736cR58
I didn't use `null` because I need to use `I18n.t` in the functional component to set the default value. If I use null I will have another check like:
```
if (!humanFormat) {
    humanFormat = "MMMM D, YYYY"
}
```

Reply:
I see.  🤔
I'm not sure if anyone else is bothered by setting the default prop to `undefined`, but maybe an alternative would be to keep `null` as the default prop and adjust formatDateProp() like :
```js
function formatDateProp(format) {
  return date && date.isValid() 
    ? moment.utc(date).format(format || I18n.t("datePicker.confirmation_format")) 
    : "";
}
```